packages/xo-server/src/api/host.js

```diff
@@ -88,6 +88,21 @@ export { restartAgent as restart_agent } // eslint-disable-line camelcase
 
 // -------------------------------------------------------------------
 
+export async function setIscsi({ host, iscsi }) {
+  return this.getXapi(host).setHostIscsi(host._xapiId, iscsi)
+}
+
+setIscsi.params = {
```

Comment:
This function set a host property, so it will have sense to be set in `host.set`.